content/post/11-Eliminating-Distractions/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Eliminating Distractions"
+date: 2010-11-13T15:56:33-07:00
+draft: true
```

Comment:
We want the new posts to be ready for publication, so please set this to false on all posts:

```suggestion
draft: false
```